src/urbit/app/chess.hoon

```diff
@@ -641,9 +642,9 @@
   |=  =path
   ^-  (unit (unit cage))
   ?+  path  (on-peek:default path)
-    ::
-    ::  read game info
-    ::  either active or archived
+  ::
+  ::  read game info
+  ::  either active or archived
```

Comment:
I think the indentation for the comments was better as before: indented to the same depth as the code which it is annotating

Reply:
Resolved. There should also be an example scry for `[%x %game @ta ~]` but I think that's added in #45.